jscomp/core/jsoo_main.ml

```diff
@@ -160,7 +160,7 @@ let dir_directory d =
 
 
 let () = 
-  dir_directory "/cmis"
+  dir_directory "/static/cmis"
```

Comment:
any reason to make such change?

Reply:
It's the other breaking change: https://github.com/ocsigen/js_of_ocaml/commit/b168b311d54c6c5e40fbf69fce798e207119aead#diff-6e61909b542c36eb53e3df028cf42e83